packages/documentation/src/content/docs/integration/deployment/docker-compose.mdx

```diff
@@ -135,8 +153,30 @@ services:
       - '3007:3007'
       - '3009:3009'
     restart: always
+```
+
+##### Environment variables
```

Comment:
For the three tables, what do you think about keeping alphabetical order but putting the required variables first?